reactioncommerce/api-plugin-inventory
# feat: add source code and enable semantic-release

This plugin replaces the internal `inventory` service.

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- see that the plugin is loaded upon startup in Reaction (use the `systemInformation` query)
- update the inventory information for a product and publish

Issue https://github.com/reactioncommerce/reaction/issues/6174

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
